apis/Google.Cloud.Logging.V2/Google.Cloud.Logging.Log4Net/GoogleStackdriverAppender.cs

```diff
@@ -289,6 +292,23 @@ private LogEntry BuildLogEntry(LoggingEvent loggingEvent)
             return logEntry;
         }
 
+        private void TryAddGitRevisionId(Dictionary<string, string> labels)
+        {
+            string gitId = null;
+            try
+            {
+                gitId = SourceContext.AppSourceContext?.Git?.RevisionId;
+            }
+            catch (Exception ex) when (ex is System.Security.SecurityException || ex is InvalidProtocolBufferException
+                || ex is InvalidJsonException || ex is UnauthorizedAccessException)
+            { }
+
+            if (!String.IsNullOrWhiteSpace(gitId))
+            {
+                labels.Add(SourceContext.GitRevisionIdLogLabel, gitId);
```

Comment:
My perception is that It's a log label for source context. It's related to both.  I feel it's acceptable to put it in either here or in the Google.Cloud.Logging.V2.dll.